apps/betterangels/src/app/(private-screens)/add-note/[noteId].tsx

```diff
@@ -222,10 +221,12 @@ export default function AddNote() {
     );
   };
 
-  const MoodAttachments = useMemo(
-    () => filterAttachments(NoteNamespaceEnum.MoodAssessment),
-    [data]
-  );
+  // TODO: Will be back with moods
+  // const MoodAttachments = useMemo(
+  //   () => filterAttachments(NoteNamespaceEnum.MoodAssessment),
+  //   [data]
+  // );
+
```

Comment:
Not blocking, but typically I'd rather see these just deleted. 
Unless there's a ticket to undo this for this sprint, there's no guarantee these things actually get put back in. Even so, this may come back in some other form etc...

Reply:
why remove the logic if we are going to bring it back a bit later and then think of implementing same logic ?

I think its better to leave as a comment then just uncomment it. We can discuss it as a parking lot item